app/assets/scripts/vendor/map.js

```diff
@@ -2,14 +2,18 @@
       // parameter when you first load the API. For example:
       // <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places">
 
+      var map;
+      var markers = [];
+
       function initMap() {
-        var map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
I needed to make the map a global variable so it could be accessed by other functions